pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/Worker.java

```diff
@@ -59,6 +59,7 @@
     private GlobalZooKeeperCache globalZkCache;
     private ConfigurationCacheService configurationCacheService;
     private final ErrorNotifier errorNotifier;
+    private static boolean isInitialDlogMetadataByPulsarTool = false;
```

Comment:
How is this flag set?

Reply:
@sijie Do you mean add a flag to config file?

Reply:
I mean how this flag is used. I don't see which method set the flag.

Reply:
https://github.com/apache/pulsar/pull/8781/files/4f5d8d5ce69a2b8f0efa2f4a72a39d5ca3cbc62a#diff-6ea96d1db19eb1c438ee988cc295057393319e0c0714771538e4ae9c7391cb86R87